plugins/chat/spec/models/chat/message_spec.rb

```diff
@@ -7,6 +7,22 @@
 
   it { is_expected.to have_many(:chat_mentions).dependent(:destroy) }
 
+  describe "validates" do
```

Comment:
We can simplify this a bit by using shoulda matchers (as you did in other files) :slightly_smiling_face: 
```ruby
  describe "Validations" do
    subject(:message) { described_class.new(message: "") }

    it { is_expected.to validate_length_of(:cooked).is_at_most(20_000) }
  end
```
Only `message` is needed here so the other validations work and have a string as they expect (without this, the matcher fails :sweat_smile:).